lint.js

```
@@ -1,3 +1,19 @@
 const add = function(a, b) {
     return a + b;
 }
+
+
+const unique = function(array) {
+    var result = []; // 使用 var 而不是 let 或 const
+    for (var i = 0; i < array.length; i++) { // 使用 var 声明循环变量
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)